src/app.md

```diff
@@ -271,6 +272,51 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-skip-dot] #ex-file-skip-dot
+<a name="ex-file-skip-dot"></a>
+##  Recursively find all files skipping the ones starting with dot
+
+[`WalkDir::filter_entry`] and [`std::iter::Iterator::filter`] qctuqlly make iterators that return only the elements that pass the closure you give to them, the difference is that [`WalkDir::filter_entry`] is specialized to handle recursive directory traversal and it will also skip whole directories if the closure is not meet.
+
+[`WalkDir::depth`] is used to skip the level 0 which is the current directory `"."`.
+
+[![walkdir-badge]][walkdir] [![cat-filesystem-badge]][cat-filesystem]
+
+```rust,no_run
+# #[macro_use]
+# extern crate error_chain;
+extern crate walkdir;
+
+use walkdir::{DirEntry, WalkDir, WalkDirIterator};
+#
+# error_chain! {
+#     foreign_links {
+#         WalkDir(walkdir::Error);
+#         Io(std::io::Error);
+#         SystemTime(std::time::SystemTimeError);
```

Comment:
Well, I dont know if the compiler say to me that Im having more than the needed ones? anyway I jst deleted SystemTime